docs/stable/cli/cli_api.md

```diff
@@ -43,7 +43,7 @@ After setting up the ServerlessLLM cluster, you can use the commands listed belo
     ```
 
 ### sllm-cli deploy
-Deploy a model using a configuration file or model name.
+Deploy a model using a configuration file or model name, with options to overwrite default configurations. The configuration file can now be incomplete, with missing sections being filled in by the default configuration.
```

Comment:
Could rephrase to: The configuration file requires minimal specifications. Sensible defaults are specified for advanced configuration options. Then you can reference the section in the docs where user can read the advanced configuration options and their default values.